### ISSUE TITLE
Problem with match_list 

### ISSUE THREAD

**Issue by iamnikiyo** [meraki-analytics/cassiopeia#67]
I have a issue with the summoner.match_list(), when i try to do:
`summoner = riotapi.get_summoner_by_name("bynikiyo")`
`match_list = summoner.match_list()`
`for participant in match_list[0].match():`
`print(participant.champion.name)`

When I obtain this match is not the last match I played, i don't know the problem.
How can I get the last match?

**Comment by sbrodehl** [comment]
Not all matches are shown, only some game modes if I remember correctly.
What type of game is missing?

Are the matches available via the [Riot  API](https://developer.riotgames.com/)?

**Comment by iamnikiyo** [comment]
Normal Games, i think it can be a problem with the 6.9 patch? The last game, which appear in the list is my last game on 6.8.

**Comment by blackpan2** [comment]
Normal games do not appear at all in the Riot API. I am including a post to the Developer forums that talks about why this was done.  The thing to look at is the first comment (from Riot employee "MattEnth")
[Dev Forum Post](https://developer.riotgames.com/discussion/community-discussion/show/Uk2kEo3E)

**Comment by sbrodehl** [comment]
So there are two different endpoints one can querry for this information.
First is `matchlist` [API](https://developer.riotgames.com/api/methods#!/1069/3683) which gives you the modes `TEAM_BUILDER_DRAFT_RANKED_5x5, RANKED_SOLO_5x5, RANKED_TEAM_3x3, RANKED_TEAM_5x5`
This is what cass returns if you retrieve `summoner.match_list()`

The second option is `game`[API](https://developer.riotgames.com/api/methods#!/1078/3718) which gives you _recent game information_ and the modes `CLASSIC, ODIN, ARAM, TUTORIAL, ONEFORALL, ASCENSION, FIRSTBLOOD, KINGPORO` and the types `CUSTOM_GAME, MATCHED_GAME, TUTORIAL_GAME`.
Problem here is, that only the _recent_ matches are available.
This is what cass returns if you retrieve `summoner.recent_games()`

One note from the Dev Forum Post above:

> TL;DR is that Normals data is restricted between friends only (and will soon be heavily enforced). We consider Ranked data to be public knowledge to all, an experience you opt-in to by playing Ranked, and have no intentions at this time to hide specific user datasets. (Riot Sargonas (NA))

**Comment by iamnikiyo** [comment]
I know about the two ways to obtain the games, I tried the summoner.recent_games() before tried summoner.match_list(). But I have other question, I need to take summoner spell game information, and in the summoner.recent_games(), which gave me a Game object has a list of participants but this list is a cassiopeia.type.core.game.Participant or a cassiopeia.type.core.Match.Participant ?
The game.participant doesn't have spell information.
